time_tracker_api/activities/activities_model.py

```diff
@@ -100,6 +133,25 @@ def get_all_with_id_in_list(
             activity_ids,
         )
 
+    def get_all(
+        self,
+        conditions: dict = None,
+        activities_id: List = None,
+        max_count=None,
+        visible_only=True,
+    ) -> list:
+        event_ctx = self.create_event_context("read-many")
+        max_count = self.repository.get_page_size_or(max_count)
```

Comment:
I think max count can be obtained inside find_all method.

Reply:
It is recommendable to use as arguments in find_all, because there are calls to activities find_all in other parts of the application (like time-entries) that already have variables or arguments with the required values.